packages/cra-template/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@reapit/cra-template",
+  "version": "0.1.0",
+  "private": true,
+  "main": "./template.json"
```

Comment:
Can we remove `private`?
And I think you should add a few more `scripts` to `formatting, linting,...`
Ref: https://github.com/reduxjs/cra-template-redux/blob/master/package.json

Reply:
sure. Thanks for suggestions. Still working on this.